trustregistry/crud.py

```diff
@@ -50,6 +50,22 @@ def get_actor_by_id(db_session: Session, actor_id: str) -> db.Actor:
     return result
 
 
+def get_actor_by_name(db_session: Session, actor_name: str) -> db.Actor:
+    bound_logger = logger.bind(body={"actor_name": actor_name})
+    bound_logger.info("Query actor by name")
+    result = (
+        db_session.query(db.Actor).filter(db.Actor.name == actor_name).one_or_none()
+    )
+
+    if result:
+        bound_logger.info("Successfully retrieved actor from database")
+    else:
+        bound_logger.info("Actor name not found")
+        raise ActorDoesNotExistException
+
+    return result
+
+
```

Comment:
Btw, not related to this function, but I can't add comment on the actual line:
```
        else:
            bound_logger.info("Bad request: {}", constraint_violation)
```
at the end of `create_actor`. 
I notice that does not raise an exception, in which case it would return None - in the unlikely case that constraint violation isn't matched by the other ifs. It should just return a generic "Bad request: Unique constraint violated - {}" with the constraint violation printed.

Also that final log message `bound_logger.info("Bad request: {}", constraint_violation)` should be error level, because it should never happen and means somethings changed.